src/kwio.sh

```diff
@@ -110,10 +111,25 @@ function success()
 # yourself in the code.
 function ask_yN()
 {
-  local message="$*"
+  local message="$1"
+  local default="${2:-"n"}"
+  local yes
+  local no
 
-  read -r -p "$message [y/N] " response
-  if [[ "$response" =~ ^([yY][eE][sS]|[yY])+$ ]]; then
+  if [[ "$default" =~ ^([yY]|[yY][eE][sS])+$ ]]; then
+    default='1'
+    yes="Y"
+    no="n"
+  else
+    default='0'
+    yes="y"
+    no="N"
+  fi
```

Comment:
You can probably remove this `else` by setting this value as the default when you declare it. For example:
```
local yes='y'
local no='n'
```

Reply:
I liked the idea, but we found a problem removing this `else`, because the `default_option` integer value is set in this block and before it has the 'y' or 'n' value. We thought that we could create a new variable to assume the integer value, while `default_option` remains only with the char given to the function, but this function would be with five local variables already. What is the best solution?